app/assets/stylesheets/style.scss

```diff
@@ -19,8 +20,15 @@
 
 .search-result-title {
   font-weight: 700;
+  font-size: 19px
   a {
     text-decoration: none;
   }
 }
 
+
+@media (min-width: 641px) {
```

Comment:
Should leave the definition of width to the predefined elements... e.g. 
@media (min-width: 641px)
....
@include media(mobile) {